tljh/config_schema.py

```diff
@@ -79,7 +79,15 @@
             "description": "User CPU and memory limits.",
             "type": "object",
             "additionalProperties": False,
-            "properties": {"memory": {"type": "string"}, "cpu": {"type": "integer"}},
+            "properties": {
+                "memory": {"type": "string"},
+                "cpu": {
+                    "anyOf": [
+                        {"type": "number", "minimum": 0},
+                        {"type": "string", "enum": ["None"]},
```

Comment:
Thanks!

Shouldn't this one be actual `null`, not the string 'None'?

```suggestion
                        {"type": "null"},
```

Reply:
Ah, looks like this is slightly more complicated than I thought!

> Setting it to `None` disables CPU limits.

I used a string `"None"` based on a quick glance at the above [docs](https://tljh.jupyter.org/en/latest/topic/tljh-config.html#user-server-limits), not realizing that moving between `tljh-config` to Python to YAML and back again might cause issues.

If I use your recommended change, the following commands both fail validation:

```bash
tljh-config set limits.cpu None
tljh-config set limits.cpu null
```

Seems as if a change to `config.py` will also need to be made to convert the string `"None"`, `"null"`, or some other placeholder-type string value passed to `tljh-config` to Python's `None` for validation.

Since most users use `tljh-config` and the docs already reference `None`, maybe `None` is the right value for this, ie users will run 
```bash
tljh-config set limits.cpu None
```
and `config.py` will handle making it Python's `None`, and then your recommended change should work.

I'm sure there are other ways to handle this as well. Let me know what you think, @minrk.

Reply:
@minrk looks like the `parse_value` method in `tljh/config.py` had logic to handle something like this but was a bit off. It checked `if value_str is None`, but there isn't a way to pass Python `None` directly to `tljh-config`. I updated that and the relevant tests.